### PR TITLE
Add a single system test

### PR DESCRIPTION
**Pull request by pjaspers**
This adds support for [Rails's new System Tests](https://guides.rubyonrails.org/testing.html#system-testing), it basically starts the Rails application and then visits the Rails Admin page. If that loads we consider it a success. This does mean that the only thing that this test will currently detect is if we somehow broke Rails Admin.

Aside from that check we also gain:

- Setup for system tests (so adding a new one if easy, but this does *not* mean that everything should have a system test, system tests are slow and should be used wisely)
- System tests will run on Travis CI
- Rspec can now run a bit more selectively

```
bundle exec rspec # runs them all
bundle exec rails spec # Runs all but the system test
bundle exec rails spec:system # Only runs the system tests
```

On CI, it will first run everything without the system tests and only if that succeeds will it run the (more expensive) system tests.

## Self proof reading checklist

- [x] Did I run Pronto and fixed all warnings
- [x] Make sure all naming and code will remain understandable in 6 month by someone new to the project or by you.
- [x] Did I test the right thing?

Thanks!

### CODE REVIEW

**Comment by mestachs** [comment]
installed chrome-driver

```
CHROME_DRIVER_VERSION=`curl -sS chromedriver.storage.googleapis.com/LATEST_RELEASE`
wget -N http://chromedriver.storage.googleapis.com/$CHROME_DRIVER_VERSION/chromedriver_linux64.zip -P ~/
unzip ~/chromedriver_linux64.zip -d ~/
rm ~/chromedriver_linux64.zip
sudo mv -f ~/chromedriver /usr/local/bin/chromedriver
sudo chown root:root /usr/local/bin/chromedriver
sudo chmod 0755 /usr/local/bin/chromedriver
```

it seems that bundle exec rails spec:system
run twice the tests
can we have this chrome driver installation in bootstrap script ?

**Comment by mestachs** [comment]
or is the sudo apt-get install chromium-chromedriver the correct way to install it ?

File: script/cibuild [hunk 1]
```diff
@@ -8,6 +8,7 @@ date "+%H:%M:%S"
 
 export RAILS_ENV="test"
 export RACK_ENV="test"
+export CI="CIJOE"
```
Comment (borisrorsvort) [inline script/cibuild]: 5gw0vwgbgnm8w

Comment (pjaspers) [reply script/cibuild]: (Stole it from here: https://github.com/defunkt/cijoe)

**Pull request merged**